src/upscaler.py

```diff
@@ -703,7 +706,16 @@ def run(self):
 
                 # downscale frames with Lanczos
                 Avalon.info(_('Lanczos downscaling frames'))
-                shutil.rmtree(self.extracted_frames)
+                try:
+                    shutil.rmtree(self.extracted_frames)
+                except Exception:
+                    if platform.system() == 'Windows':
+                        for file in self.extracted_frames.iterdir():
+                            Avalon.info(_("Remove file:[%s] readOnly attribute." % file))
+                            os.chmod(file, stat.S_IWRITE)
+
+                        shutil.rmtree(self.extracted_frames)
```

Comment:
What if it's not windows? You need to throw the exception again, otherwise it will keep on going ignoring the errors.